docs/resources/equinix_fabric_connection.md

```diff
@@ -192,10 +199,6 @@ Optional:
 - `type` (String) Virtual Device type
 - `uuid` (String) Equinix-assigned Virtual Device identifier
 
-Read-Only:
-
-- `href` (String) Unique Resource Identifier
```

Comment:
Is href not supported for VD schema? I see in the response schema that it will be returned. Might want to check into this a bit deeper.

Reply:
It's supported for the schema, but it wasn't used in model so I removed it. But I will add it.

Reply:
Same goes for here; we just need the provider to be very consistent with what is being exposed in its schema and in the documentation. It doesn't always have to match the 4.10 spec exactly, but the Terraform Docs/Schema/Internal Parsing and sending in API requests needs to be consistent.